src/components/ManageAccountsDrawer/ManageAccountsDrawer.tsx

```diff
@@ -19,11 +22,12 @@ export const ManageAccountsDrawer = ({
   onClose,
   chainId: parentSelectedChainId,
 }: ManageAccountsDrawerProps) => {
+  const wallet = useWallet().state.wallet
   const [step, setStep] = useState<ManageAccountsStep>('selectChain')
   const [selectedChainId, setSelectedChainId] = useState<ChainId | null>(null)
 
   // TODO: Implement Ledger specific logic
-  const isLedger = false
+  const isLedgerWallet = wallet ? isLedger(wallet) : false
```

Comment:
Since we only consume this in `handleNext`, we may want to early return there `if (!wallet)` instead

Reply:
Sounds like a plan: https://github.com/shapeshift/web/pull/6779/commits/fa0ec15b703f1d063bdc1e91a7689a4d7ca79295